scico/linop/radon_svmbir.py

```diff
@@ -11,41 +11,52 @@
 `svmbir <https://github.com/cabouman/svmbir>`_ package.
 """
 
+from typing import Optional
+
 import numpy as np
 
 import jax
 import jax.experimental.host_callback
 
-try:
-    import svmbir
-except ImportError:
-    raise ImportError(
-        "Could not import svmbir, please refer to INSTALL.rst "
-        "for instructions on how to install the SVMBIR."
-    )
-
 import scico.numpy as snp
 from scico.loss import WeightedSquaredL2Loss
 from scico.typing import JaxArray, Shape
 
 from ._linop import LinearOperator
 
+try:
+    import svmbir
+except ImportError:
+    raise ImportError("Could not import svmbir; please install it.")
+
 
 class ParallelBeamProjector(LinearOperator):
-    r"""Parallel beam projector based on svmbir."""
+    r"""Parallel beam Radon transform based on svmbir.
+
+    Perform tomographic projection of an image at specified angles,
+    using the `svmbir <https://github.com/cabouman/svmbir>`_ package. The
+    ``is_masked`` option defines a valid region for projection and updates.
+    Pixels outside the valid region are not projected or updated during
+    a reconstruction.
+    """
```

Comment:
It's not clear what `updates` and `reconstruction` here refer too, since this class is just a Radon transform. Presumably it refers to the when the `ParallelBeamProjector` is used with `SVMBIRWeightedSquaredL2Loss`? If so, this should be clarified here.

Reply:
Clarified in commit 219e5ed